specification/04-requirements.adoc

```diff
@@ -177,24 +177,33 @@ IDE streams.
 When setting IDE keys into a CoVE-IO device, the TSM relies on the DSM IDE Key
 Management (`IDE_KM`) support, and its ability to receive IDE_KM messages over a
 Secured SPDM session. However, there are no architecturally-defined PCIe
-protocol for managing Root Port IDE keys.
-
-Instead of adding multiple vendor-specific `IDE_KM` implementations to the TSM,
-the TSM relies on the platform Root-of-Trust (ROT) to implement the `IDE_KM`
-protocol and abstract the platform specific PCIe RP implementation away from
-the TSM. The TSM establishes a Secured SPDM session with the ROT over a host
-accessible DOE mailbox, and then sets platform RP IDE keys over that session.
+protocol for managing root port IDE keys and this specification considers two
+variants for conveying IDE keys from the TSM to the PCIe root port.
+
+In one variant the platform provides vendor-specific protection for the
+communication path between these entities. Here the TSM may program IDE keys
+directly through the `IDE_KM` protocol or choose any other vendor-specific
+mechanism.
+
+In the second variant this protection is not provided by the platform and/or
+PCIe root ports are not directly accessible to the TSM. However, if the platform
+Root-of-Trust (RoT) implements the `IDE_KM` protocol, the TSM should establish a
+Secured SPDM session with the RoT over a a host accessible DOE mailbox. It can
+then sets platform RP IDE keys over that session. Having a PCIe accessible and
+`IDE_KM` capable RoT on the platform abstracts the vendor-specific PCIe root
```

Comment:
@sameo : Thanks for the update. Looks good to me.
Two minor typos observed: 
- 191: 'a a '
- 191/192: "It can then set" or "It then sets"
-

Reply:
Good catch, typos are fixed now.